src/Chapter09/Listing09.20.ImplementingValueEqualityOnCustomTypes.cs

```diff
@@ -0,0 +1,65 @@
+using System;
+using System.Collections.Generic;
+
+namespace AddisonWesley.Michaelis.EssentialCSharp.Chapter09.Listing09_20
+{
+    public class Foo : IEquality<Foo>
```

Comment:
Do the examples use `IEquality<T>`? I would have expected them to use the built in `IEquatable<T>`

Reply:
Yes, the book says to implement the `IEquality<T>` interface. There's a footnote attached to it that says it is a generic interface discussed in Chapter 12. I looked through Chapter 12 and couldn't find any mention of `IEquality<T>`, so I just made something of my own. Thoughts?